website/docs/getting-started/configuration.md

```diff
@@ -77,8 +77,8 @@ For all available command line settings, run `GarnetServer.exe -h` or `GarnetSer
 | **IndexSize** | ```-i```<br/>```--index``` | ```string``` | Memory size | Size of hash index in bytes (rounds down to power of 2) |
 | **IndexMaxSize** | ```--index-max-size``` | ```string``` | Memory size | Max size of hash index in bytes (rounds down to power of 2) |
 | **MutablePercent** | ```--mutable-percent``` | ```int``` |  | Percentage of log memory that is kept mutable |
-| **ObjectStoreTotalMemorySize** | ```--obj-total-memory``` | ```string``` | Memory size | Total object store log memory used including heap memory in bytes |
-| **ObjectStoreLogMemorySize** | ```--obj-memory``` | ```string``` | Memory size | Object store log memory used in bytes excluding heap memory |
+| **ObjectStoreHeapMemorySize** | ```--obj-heap-memory``` | ```string``` | Memory size | Object store heap memory size in bytes |
+| **ObjectStoreLogMemorySize** | ```--obj-log-memory``` | ```string``` | Memory size | Object store log memory used in bytes excluding heap memory |
```

Comment:
Object store log memory used in bytes (only the log with references, excluding heap memory of objects pointed to from the log)